.github/workflows/code_check.yml

```diff
@@ -0,0 +1,72 @@
+name: Code Checks
+on:
+  - push
+  - pull_request
+  - workflow_dispatch
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
```

Comment:
Isn't code quality check using PEP8 standard independent of python version?

Reply:
Moved code check for linting over to pre-commit CI. We just need to enable it by giving it proper permissions to the repo.